### PR TITLE
add prometheus metric exporter support for nodejs autoinstrumentation

### PR DESCRIPTION
**Pull request by jakobjoachim**
The Autoinstrumentation Image only supports the OTLPMetricExporter metrics exporter until now. 

This reads the environment variable OTEL_METRICS_EXPORTER and uses the configured one or throws and error if anything else is set.

### CODE REVIEW

**Comment by jaronoff97** [comment]
Hello, thank you for contribution! can you be sure to fill out the CLA? Also, if you could add in an e2e test for nodejs autoinstrumentation that would be great.

**Comment by jakobjoachim** [comment]
> Hello, thank you for contribution! can you be sure to fill out the CLA? Also, if you could add in an e2e test for nodejs autoinstrumentation that would be great.

Should I add the test to the existing "instrumentation-nodejs-multicontainer" test or create a new one for it?

**Comment by jaronoff97** [comment]
hmm maybe add it to the existing one.

**Review by TylerHelmuth — COMMENTED** [review]
@jakobjoachim please add a changelog entry for this by running `make chlog-new` from the root of the repository.

**Review by jaronoff97 — APPROVED** [review]

File: .chloggen/read_env_vars_for_nodejs_autoinstrumentation.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: nodejs autoinstrumentation
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "prometheus metric exporter support for nodejs autoinstrumentation"
+
+# One or more tracking issues related to the change
+issues: []
```
Comment (TylerHelmuth) [inline .chloggen/read_env_vars_for_nodejs_autoinstrumentation.yaml:11]:
```suggestion
issues: [1798]
```

**Comment by jakobjoachim** [comment]
The test that is failing is the `instrumentation-apache-multicontainer` test. Is that a flaky test by any chance?

**Comment by jakobjoachim** [comment]
I agree that there should be a `getMetricReaderFromEnv()` or a similar function in `@opentelemetry/sdk-node`. 
Same as the `getNodeAutoInstrumentations()` function that exists already.
The same thing is also missing for logging and tracing. 
The question is if this is a good workaround until such a function is implemented or if we move this code over there directly.

**Comment by TylerHelmuth** [comment]
I would prefer this workaround for now.  That lets operator users get the functionality they want, and then updating this code once opentelemetry-js supports this feature is a non-breaking change to operator end-users.

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**